ykrt/src/compile/jitc_yk/aot_ir.rs

```diff
@@ -711,18 +711,23 @@ pub(crate) fn const_int_bytes_to_str(num_bits: u32, bytes: &[u8]) -> String {
     }
 }
 
-/// A fixed-width integer type.
+/// A fixed-width integer.
 ///
-/// Signedness is not specified. Interpretation of the bit pattern is delegated to operations upon
-/// the integer.
+/// Note:
+///   1. These integers range in size from 1..23 bits (inclusive). This is inherited [from LLVM's
```

Comment:
1..2^{23}, not 1..23!

You had me really worried for a moment there! Also commit message needs updating.

Reply:
Hmm, you're right that what I wrote is really easily misread. Mea culpa! Fixed in 187843ec.